css/index.css

```diff
@@ -143,4 +144,113 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+header {
+    display:flex;
+    justify-content: flex-start;
+    align-items: baseline;
+    flex-flow: row wrap;
+    padding-bottom: 4%;
+    padding-top: 4%;
+}
+
+header a {
+    text-decoration: none;
+    color: black;
+    padding: 5%;
+}
+
+header .header-image {
+    width: 115%;
+}
+
```

Comment:
Good job allowing the parent to style the boxes and just overriding the color.  You can remove the class="box" and just use your ids.

Reply:
I am pretty sure that the classes were a result of me fiddling trying to make the colors work.

Reply:
There are multiple ways to do this, the important thing is to remember how CSS applies precedence.  Another way to do this without adding another class or id is to use a CSS selector :nth-child() to apply the colors.

Reply:
I first tried that but I couldn't get it to work for me.

Reply:
it would be .boxes .box:nth-child(1) for the first box or for the first box is could also be .boxes .box:first-child().  I would have to take a look at your project again because it does matter exactly how you laid out your CSS/classes.

Reply:
I will try and nail down that on the next project! Thank you for your input.